chains/evm/cli/flags/utils.go

```diff
@@ -58,3 +60,23 @@ func defineSender(cmd *cobra.Command) (*secp256k1.Keypair, error) {
 	var AliceKp = keystore.TestKeyRing.EthereumKeys[keystore.AliceKey]
 	return AliceKp, nil
 }
+
+func ProcessResourceID(resourceID string) ([32]byte, error) {
+	if resourceID[0:2] == "0x" {
+		resourceID = resourceID[2:]
+	}
+	resourceIdBytes, err := hex.DecodeString(resourceID)
+	if err != nil {
+		return [32]byte{}, err
+	}
+	return calls.SliceTo32Bytes(resourceIdBytes), nil
+}
+
+func CheckRequiredFlags(cmd *cobra.Command, flags ...string) {
```

Comment:
I think proper naming would be something like `MarkAllFlagsAsRequired` because from original name I would assume that this functions checks if flags are required.